integrated_channels/blackboard/migrations/0001_initial_squashed_0011_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id.py

```diff
@@ -0,0 +1,131 @@
+# Generated by Django 3.2.11 on 2022-01-27 11:11
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.migrations.operations.special
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+import simple_history.models
+import uuid
+
+
+# NOTE: the replaced migrations should not be deleted until after nutmeg.
+
+# Functions from the following migrations need manual copying.
+# Move them and any dependencies into this file, then update the
+# RunPython operations to refer to the local versions:
+# integrated_channels.blackboard.migrations.0010_auto_20211221_1532
+
```

Comment:
Was this just about `create_uuid`? If so, you can remove this comment since this manual step is complete.

Reply:
> Thanks. I think this is mostly ready.
> 
> 1. Just one minor comment that I think just needs to be deleted.
> 2. I imagine you need to bump the version and update the CHANGELOG as well.

I just asked owners for the review. After their review I will bump the version. Would you please share your thoughts on running these migration on prod ? One table has 6 million and other has 2 million records.

Reply:
1. Read the section of https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-CheckingSQLforamigration on Checking SQL. Shouldn't this just be dropping a constraint?
2. Are you concerned if they want to add back the constraint, or about some other migration? I forgot you may want to discuss with them whether or not they want the constraint added back or not.

Am I answering your concern, or have I misunderstood something?

Reply:
Thanks for document.
1. It's just dropping a constraint but want to double check with SRE because table has lots of records. Owning team review is important may be they have some different plan for this problem.
2. Comment removed.